modules/base/variables.tf

```diff
@@ -13,6 +13,12 @@ variable "region" {
   description = "GCP Region to provision resources in."
 }
 
+variable "humanitec_org_id" {
+  description = "Humanitec Organization ID (required for Backstage)"
```

Comment:
This documentation is outdated now, right? The Organization ID is now required in any case.

(Same for for the root `variables.tf`)

Reply:
Fixed both!